netty/src/main/java/io/grpc/netty/NettyServerBuilder.java

```diff
@@ -247,6 +249,18 @@ protected NettyServer buildTransportServer() {
   }
 
   @Override
+  public NettyServerBuilder useTransportSecurity(TrustManagerFactory trustManagerFactory,
+                                                 KeyManagerFactory keyManagerFactory) {
+    try {
+      sslContext = GrpcSslContexts.forServer(trustManagerFactory, keyManagerFactory).build();
+    } catch (SSLException e) {
+      // This should likely be some other, easier to catch exception.
```

Comment:
No worries. `IllegalArgumentExceptions` sounds good. I've added new commit updating to it.